README.md

```
@@ -1,64 +1,64 @@
-    <picture>
-      <img src="https://cdn.scalekit.cloud/v1/scalekit-logo-dark.svg" height="64">
-    </picture>
-  </a>
-  <br/>
-</p>
+<div align="center">
 
-# Official Java SDK
+<a href="https://scalekit.com" target="_blank" rel="noopener noreferrer">
+  <picture>
+    <img src="https://cdn.scalekit.cloud/v1/scalekit-logo-dark.svg" height="64">
```

markdownlint-cli2 (0.22.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)